docs/website/theory/theoryOverview.md

```diff
@@ -15,18 +15,16 @@ explained here with the example of a car.
 
 ### Interface of a car
 Let's first define the interface of a car. A car is from a certain brand and it 
-has a maximum velocity. The interface of a car is then (written as a C++20 
-concept):
+has a maximum velocity. The interface of a car is then:
+
+- `brand()`: a function which returns the brand as a string
+- `maxvelocity()`: a function which returns the maximum velocity as a double

```

Comment:
Sollen wir wirklich anfangen solche C++ Details zu erklären?
Man könnte im Code auch nur die Schlagworte oder das passende Idiom nennen das man nutzt.
Wir können hier auch eine Best-Pratice liste machen wie man Klassen schreibt und da dann auf die Literatur verweisen. Hier könnte man das dann unterfüttern mit sachen wie https://youtu.be/xnqTKD8uD64?t=3559.

Reply:
Die Erklärung habe ich eingefügt, um bei der Erklärung der Interfaces nicht jedes mal schreiben zu müssen, ob es sich um eine Member-Funktion oder free function handelt. Deshalb habe ich das einmal als Konvention definiert, wenn das worum es geht (in diesem Fall car) in der Klammer steht, bedeutet es, dass das Interface eine free function fordert. Ich hoffe das kam einigermaßen rüber, ansonsten kann ich es nochmal präzisieren.

Allgemein bin ich auch dagegen, in der Dokumentation C++ zu erklären.